x/slashing/keeper/params.go

```diff
@@ -50,10 +50,6 @@ func (k Keeper) GetParams(ctx sdk.Context) (params types.Params) {
 
 // SetParams sets the x/slashing module parameters.
 func (k Keeper) SetParams(ctx sdk.Context, params types.Params) error {
-	if err := params.Validate(); err != nil {
```

Comment:
The change for this is mentioned in a changelog from this PR: https://github.com/cosmos/cosmos-sdk/pull/15786.
This makes consistent all `SetParams` methods, some were validating params whereas some did not.
Now we always validate in the msg server and only there.

Reply:
should we make setParams internal to avoid others accidentally calling it without calling validation or leave a contact comment saying the caller should call validate

Reply:
You may have a module that updates other module params, so making it private is a no-go imho.
We _could_ add a comment, indeed.